### PR TITLE
Update the Readme summary and typo in Lorenz notebook

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,3 +146,18 @@ over the set of tested equations and some specific examples may differ.
...
+
+### Parameter Estimation
+
+- Broadly two different approaches have been used, Bayesian Inference and Optimisation 
+  algorithms.
+- In general it seems that the optimisation algorithms perform more accurately but that can be 
+  attributed to the larger number of data points being used in the optimisation cases, Bayesian 
+  approach tends to be slower of the two and hence lesser data points are used, accuracy can 
+  increase if proper data is used.
+- Within the different available optimisation algorithms, BBO, GN_CRS2_LM, LD_SLSQP, LN_BOBYQA
```
Comment (ChrisRackauckas) [inline README.md]: Name which packages these are from, and mention the difference between global and local.

File: README.md [hunk 2]
```diff
@@ -146,3 +146,18 @@ over the set of tested equations and some specific examples may differ.
...
+- Broadly two different approaches have been used, Bayesian Inference and Optimisation 
+  algorithms.
+- In general it seems that the optimisation algorithms perform more accurately but that can be 
+  attributed to the larger number of data points being used in the optimisation cases, Bayesian 
+  approach tends to be slower of the two and hence lesser data points are used, accuracy can 
+  increase if proper data is used.
+- Within the different available optimisation algorithms, BBO, GN_CRS2_LM, LD_SLSQP, LN_BOBYQA 
+  and LN_NELDERMEAD perform the best.
+- The different backends options available for Bayesian method offer some tradeoffs beteween
+  time, accuracy and control. It is observed that sufficiently high accuracy can be observed with
```
Comment (ChrisRackauckas) [inline README.md]: Do we have enough information to suggest heuristics yet?

Comment (Vaibhavdixit02) [reply README.md]: That is true, but at face value one can say something about the general trend though?

Comment (ChrisRackauckas) [reply README.md]: Oh no, I meant do we have enough information to suggest an in-category heuristic trend? Which of the 3 are generally faster, or more accurate?

Comment (Vaibhavdixit02) [reply README.md]: For faster we have the time from the `@time` macro and for accuracy I am just using an euclidean of sorts in my head I would say. I get that it isn't a concrete statement this way, would you prefer no mention of specific algorithms?

File: README.md [hunk 3]
```diff
@@ -146,3 +146,21 @@ over the set of tested equations and some specific examples may differ.
...
+
+- Broadly two different approaches have been used, Bayesian Inference and Optimisation 
+  algorithms.
+- In general it seems that the optimisation algorithms perform more accurately but that can be 
+  attributed to the larger number of data points being used in the optimisation cases, Bayesian 
+  approach tends to be slower of the two and hence lesser data points are used, accuracy can 
+  increase if proper data is used.
+- Within the different available optimisation algorithms, BBO from the BlackBoxOptim package and GN_CRS2_LM, 
+  LD_SLSQP,LN_BOBYQA and LN_NELDERMEAD from the NLopt package perform the best.
+- The global versus local optimization make a huge difference in the timings. BBO tends to find the correct
```
Comment (ChrisRackauckas) [inline README.md]: The global ... => The choice of global ...

File: README.md [hunk 4]
```diff
@@ -146,3 +146,21 @@ over the set of tested equations and some specific examples may differ.
...
+- Broadly two different approaches have been used, Bayesian Inference and Optimisation 
+  algorithms.
+- In general it seems that the optimisation algorithms perform more accurately but that can be 
+  attributed to the larger number of data points being used in the optimisation cases, Bayesian 
+  approach tends to be slower of the two and hence lesser data points are used, accuracy can 
+  increase if proper data is used.
+- Within the different available optimisation algorithms, BBO from the BlackBoxOptim package and GN_CRS2_LM, 
+  LD_SLSQP,LN_BOBYQA and LN_NELDERMEAD from the NLopt package perform the best.
+- The global versus local optimization make a huge difference in the timings. BBO tends to find the correct  
+  solution for a global optimization setup. For local optimization, most methods in NLopt, like :LN_BOBYQA, solve the
```
Comment (ChrisRackauckas) [inline README.md]: You should probably mention `GN_CRS2_LM` is global, while the others are local.

File: README.md [hunk 5]
```diff
@@ -146,3 +146,21 @@ over the set of tested equations and some specific examples may differ.
...
+  algorithms.
+- In general it seems that the optimisation algorithms perform more accurately but that can be 
+  attributed to the larger number of data points being used in the optimisation cases, Bayesian 
+  approach tends to be slower of the two and hence lesser data points are used, accuracy can 
+  increase if proper data is used.
+- Within the different available optimisation algorithms, BBO from the BlackBoxOptim package and GN_CRS2_LM, 
+  LD_SLSQP,LN_BOBYQA and LN_NELDERMEAD from the NLopt package perform the best.
+- The global versus local optimization make a huge difference in the timings. BBO tends to find the correct  
+  solution for a global optimization setup. For local optimization, most methods in NLopt, like :LN_BOBYQA, solve the  
+  problem very fast.
```
Comment (ChrisRackauckas) [inline README.md]: but require a good initial condition

File: README.md (hunk 2 above)
Comment (ChrisRackauckas) [reply README.md]: I would prefer a mention of what's fast and what's accurate if you think they are already starting to differentiate from each other.

**Comment by Vaibhavdixit02** [comment]
@ChrisRackauckas does it look alright now? Do you want to change the comparison in accuracy?

**Pull request merged**